README.md

```diff
@@ -110,9 +140,18 @@ self.annotator.dataSendingHandler = ^(NSArray *data, NSError *error) {
 };
 ```
 
-## Requirements
+## Contribute
 
-To be prepared to develop with the Annotations Accelerator Pack for iOS:
+If you'd like to contribute to OpenTok Accelerator Pack development, please follow the guidelines in the [contributing files](/.github).
 
-1. Install Xcode version 5 or later, with ARC enabled.
-2. Your device must be running iOS 8 or later.
+
+## License
+
+This project is under the [MIT License](./LICENSE)
+
+
+# About OpenTok
+
+![logo](./tokbox-logo.png)
```

Comment:
@PanoramicRum IMHO, we should not duplicate logo in the same Readme. Also main logo is not visible (could you review the img route?)